docs/api-docs/headless/end-to-end-guide/graphql.mdx

```diff
@@ -0,0 +1,756 @@
+    "checkoutEntityId": "812ece1a-da23-46eb-ab6b-c2ee210aae54",
+    "data": {
+      "address": {
+        "firstName": "Joe",
+        "lastName": "Blogs",
+        "email": "joe@example.com",
+        "company": "BlogIndustries",
+        "address1": "123 Yar st",
+        "address2": "",
+        "city": "RWC",
+        "stateOrProvince": "CA",
+        "stateOrProvinceCode": "CA",
+        "countryCode": "US",
+        "postalCode": "94061",
+        "phone": "6501231212",
+        "shouldSaveAddress": false
+      }
+    }
+  }
+}
+```
+</Tab>
+ <Tab>
+    
+ ```json filename="Example mutation: Add checkout billing address" showLineNumbers copy
+    {
+  "data": {
+    "checkout": {
+      "addCheckoutBillingAddress": {
+        "checkout": {
+          "entityId": "812ece1a-da23-46eb-ab6b-c2ee210aae54"
+        }
+      }
+    }
+  }
+}
+ ```
+  </Tab>
+</Tabs>
+
+
+## Add a checkout shipping consignment
+
+This mutation adds a shipping consignment to an existing checkout.
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+ ```graphql filename="Example mutation: Add checkout billing address" showLineNumbers copy
+mutation addCheckoutShippingConsignments($addCheckoutShippingConsignmentsInput: AddCheckoutShippingConsignmentsInput!) {
+  checkout {
+    addCheckoutShippingConsignments(input: $addCheckoutShippingConsignmentsInput) {
+      checkout {
+        entityId
+        shippingConsignments {
+          entityId
+          availableShippingOptions {
+            entityId
+          }
+          selectedShippingOption {
+            entityId
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+```json filename="GraphQL variables" showLineNumbers copy 
+{
+  "addCheckoutShippingConsignmentsInput": {
+    "checkoutEntityId": "812ece1a-da23-46eb-ab6b-c2ee210aae54",
+    "data": {
+      "consignments": [
+        {
+          "address": {
+            "firstName": "Joe",
+            "lastName": "Blogs",
+            "email": "joe@example.com",
+            "company": "BlogIndustries",
+            "address1": "123 Yar st",
+            "city": "RWC",
+            "stateOrProvince": "CA",
+            "stateOrProvinceCode": "CA",
+            "countryCode": "US",
+            "postalCode": "94061",
+            "phone": "6501231212",
+            "shouldSaveAddress": false
+          },
+          "lineItems": [
+            {
+              "lineItemEntityId": "0a281902-d548-4d2e-a930-dc2aae9bfc39",
+              "quantity": 1
+            }
+          ]
+        }
+      ]
+    }
+  }
+}
+```
+</Tab>
+ <Tab>
+    
+ ```json filename="Example mutation: Add a checkout shipping consignment" showLineNumbers copy
+{
+  "data": {
+    "checkout": {
+      "addCheckoutShippingConsignments": {
+        "checkout": {
+          "entityId": "812ece1a-da23-46eb-ab6b-c2ee210aae54",
+          "shippingConsignments": [
+            {
+              "entityId": "6478f628f1d8f",
+              "availableShippingOptions": [
```

Comment:
I think it's important to call out after this step that the user must note/record the ID of one off the available shipping options in the response, for the next step.

That would be a good thing to add consistently in each step (such as noting to capture the ID of the cart after the first createCart request).

Reply:
@bc-traciporter I see a note in the addCheckoutShippingConsignments step about noting and recording the consignment ID for later, but not noting and recording an ID of an available shipping method.